2013/12/02 17:30:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'usrCrediTaxi'@'localhost' (using password: YES)' in C:\xampp\htdocs\creditaxi\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\creditaxi\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\creditaxi\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\creditaxi\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\creditaxi\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\xampp\htdocs\creditaxi\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\xampp\htdocs\creditaxi\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\creditaxi\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\creditaxi\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud')
#14 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/creditaxi/gii/crud
HTTP_REFERER=http://localhost/creditaxi/gii/default/index
---
2013/12/02 21:15:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/creditaxi/login
---
2013/12/02 21:15:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/creditaxi/login
---
2013/12/04 02:12:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\crediTaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/login')
#3 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/crediTaxi/site/site/login
HTTP_REFERER=http://localhost/crediTaxi/site/login
---
2013/12/04 02:12:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\crediTaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/login')
#3 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/crediTaxi/site/site/login
HTTP_REFERER=http://localhost/crediTaxi/site/contact
---
2013/12/04 02:13:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\crediTaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/login')
#3 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/crediTaxi/site/site/login
HTTP_REFERER=http://localhost/crediTaxi/site/site/login
---
2013/12/04 02:58:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "conductorController".' in C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('conductorContro...')
#1 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/crediTaxi/conductorController
---
2013/12/04 02:59:28 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`conductor`, CONSTRAINT `FK_conduce_Taxi` FOREIGN KEY (`placa`) REFERENCES `taxi` (`placa`)). The SQL statement executed was: INSERT INTO `conductor` (`id_conductor`, `placa`) VALUES (:yp0, :yp1).
in C:\xampp\htdocs\crediTaxi\protected\controllers\ConductorController.php (73)
in C:\xampp\htdocs\crediTaxi\index.php (13)
2013/12/04 02:59:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`conductor`, CONSTRAINT `FK_conduce_Taxi` FOREIGN KEY (`placa`) REFERENCES `taxi` (`placa`)). The SQL statement executed was: INSERT INTO `conductor` (`id_conductor`, `placa`) VALUES (:yp0, :yp1)' in C:\xampp\htdocs\crediTaxi\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\crediTaxi\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\crediTaxi\protected\controllers\ConductorController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\crediTaxi\framework\web\actions\CInlineAction.php(49): ConductorController->actionCreate()
#4 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\crediTaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\crediTaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\crediTaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\crediTaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('conductor/creat...')
#14 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/crediTaxi/conductor/create
HTTP_REFERER=http://localhost/crediTaxi/conductor/create
---
2013/12/05 05:05:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\creditaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/login')
#3 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/creditaxi/site/site/login
---
2013/12/05 05:05:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\creditaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/login')
#3 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/creditaxi/site/site/login
---
---
2013/12/05 05:05:36 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('usuario/admin')
#9 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/creditaxi/usuario/admin
HTTP_REFERER=http://localhost/creditaxi/usuario
---
2013/12/05 16:11:36 [error] [php] session_regenerate_id() [<a href='function.session-regenerate-id'>function.session-regenerate-id</a>]: Cannot regenerate session id - headers already sent (C:\xampp\htdocs\crediTaxi\framework\web\CHttpSession.php:185)
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\auth\CWebUser.php(227): CWebUser->changeIdentity()
#1 C:\xampp\htdocs\crediTaxi\protected\models\LoginForm.php(71): CWebUser->login()
#2 C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php(62): LoginForm->login()
#3 C:\xampp\htdocs\crediTaxi\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\crediTaxi\index.php(13): CWebApplication->run()
REQUEST_URI=/crediTaxi/site/login
in C:\xampp\htdocs\crediTaxi\protected\models\LoginForm.php (71)
in C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php (62)
in C:\xampp\htdocs\crediTaxi\index.php (13)
2013/12/05 16:54:01 [error] [exception.CDbException] exception 'CDbException' with message 'Usuario does not have relation "Rol".' in C:\xampp\htdocs\crediTaxi\framework\db\ar\CActiveRecord.php:248
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\protected\components\UserIdentity.php(33): CActiveRecord->getRelated('Rol')
#1 C:\xampp\htdocs\crediTaxi\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\crediTaxi\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\crediTaxi\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\crediTaxi\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php(61): CModel->validate()
#6 C:\xampp\htdocs\crediTaxi\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/crediTaxi/site/login
HTTP_REFERER=http://localhost/crediTaxi/site/login
---
2013/12/05 17:38:47 [error] [exception.CException] exception 'CException' with message 'SiteController and its behaviors do not have a method or closure named "getBasicItems".' in C:\xampp\htdocs\crediTaxi\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\protected\components\MenuItems.php(7): CComponent->__call('getBasicItems', Array)
#1 C:\xampp\htdocs\crediTaxi\protected\components\MenuItems.php(7): SiteController->getBasicItems()
#2 C:\xampp\htdocs\crediTaxi\protected\views\layouts\main.php(31): MenuItems->getMenuItems('1033729279')
#3 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\crediTaxi\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\crediTaxi\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#7 C:\xampp\htdocs\crediTaxi\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#8 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(206): COutputProcessor->run()
#9 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#10 C:\xampp\htdocs\crediTaxi\protected\views\layouts\column1.php(6): CBaseController->endContent()
#11 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#12 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#14 C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php(67): CController->render('login', Array)
#15 C:\xampp\htdocs\crediTaxi\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#16 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('login')
#20 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#21 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/crediTaxi/site/login
HTTP_REFERER=http://localhost/crediTaxi/site/login
---
2013/12/05 17:39:40 [error] [php] array_values() expects parameter 1 to be array, object given (C:\xampp\htdocs\crediTaxi\framework\zii\widgets\CMenu.php:295)
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(147): CMenu->init()
#1 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(172): SiteController->createWidget()
#2 C:\xampp\htdocs\crediTaxi\protected\views\layouts\main.php(35): SiteController->widget()
#3 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(126): require()
#4 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(95): SiteController->renderInternal()
#5 C:\xampp\htdocs\crediTaxi\framework\web\widgets\CContentDecorator.php(76): SiteController->renderFile()
#6 C:\xampp\htdocs\crediTaxi\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#7 C:\xampp\htdocs\crediTaxi\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#8 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(206): CContentDecorator->run()
#9 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(300): SiteController->endWidget()
#10 C:\xampp\htdocs\crediTaxi\protected\views\layouts\column1.php(6): SiteController->endContent()
#11 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(126): require()
#12 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(95): SiteController->renderInternal()
#13 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(784): SiteController->renderFile()
#14 C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php(67): SiteController->render()
#15 C:\xampp\htdocs\crediTaxi\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#16 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(308): CInlineAction->runWithParams()
#17 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(286): SiteController->runAction()
#18 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(265): SiteController->runActionWithFilters()
#19 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): SiteController->run()
#20 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController()
#21 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\crediTaxi\index.php(13): CWebApplication->run()
REQUEST_URI=/crediTaxi/site/login
in C:\xampp\htdocs\crediTaxi\protected\views\layouts\main.php (35)
in C:\xampp\htdocs\crediTaxi\protected\views\layouts\column1.php (6)
in C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php (67)
2013/12/05 19:43:04 [error] [php] Invalid argument supplied for foreach() (C:\xampp\htdocs\crediTaxi\framework\zii\widgets\CMenu.php:260)
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\zii\widgets\CMenu.php(156): CMenu->normalizeItems()
#1 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(147): CMenu->init()
#2 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(172): SiteController->createWidget()
#3 C:\xampp\htdocs\crediTaxi\protected\views\layouts\main.php(35): SiteController->widget()
#4 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(126): require()
#5 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(95): SiteController->renderInternal()
#6 C:\xampp\htdocs\crediTaxi\framework\web\widgets\CContentDecorator.php(76): SiteController->renderFile()
#7 C:\xampp\htdocs\crediTaxi\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#8 C:\xampp\htdocs\crediTaxi\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#9 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(206): CContentDecorator->run()
#10 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(300): SiteController->endWidget()
#11 C:\xampp\htdocs\crediTaxi\protected\views\layouts\column1.php(6): SiteController->endContent()
#12 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(126): require()
#13 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(95): SiteController->renderInternal()
#14 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(784): SiteController->renderFile()
#15 C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php(67): SiteController->render()
#16 C:\xampp\htdocs\crediTaxi\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#17 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(308): CInlineAction->runWithParams()
#18 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(286): SiteController->runAction()
#19 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(265): SiteController->runActionWithFilters()
#20 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): SiteController->run()
#21 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController()
#22 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#23 C:\xampp\htdocs\crediTaxi\index.php(13): CWebApplication->run()
REQUEST_URI=/crediTaxi/site/login
in C:\xampp\htdocs\crediTaxi\protected\views\layouts\main.php (35)
in C:\xampp\htdocs\crediTaxi\protected\views\layouts\column1.php (6)
in C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php (67)
2013/12/05 19:43:56 [error] [php] Invalid argument supplied for foreach() (C:\xampp\htdocs\crediTaxi\framework\zii\widgets\CMenu.php:260)
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\zii\widgets\CMenu.php(156): CMenu->normalizeItems()
#1 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(147): CMenu->init()
#2 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(172): SiteController->createWidget()
#3 C:\xampp\htdocs\crediTaxi\protected\views\layouts\main.php(35): SiteController->widget()
#4 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(126): require()
#5 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(95): SiteController->renderInternal()
#6 C:\xampp\htdocs\crediTaxi\framework\web\widgets\CContentDecorator.php(76): SiteController->renderFile()
#7 C:\xampp\htdocs\crediTaxi\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#8 C:\xampp\htdocs\crediTaxi\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#9 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(206): CContentDecorator->run()
#10 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(300): SiteController->endWidget()
#11 C:\xampp\htdocs\crediTaxi\protected\views\layouts\column1.php(6): SiteController->endContent()
#12 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(126): require()
#13 C:\xampp\htdocs\crediTaxi\framework\web\CBaseController.php(95): SiteController->renderInternal()
#14 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(784): SiteController->renderFile()
#15 C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php(67): SiteController->render()
#16 C:\xampp\htdocs\crediTaxi\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#17 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(308): CInlineAction->runWithParams()
#18 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(286): SiteController->runAction()
#19 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(265): SiteController->runActionWithFilters()
#20 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): SiteController->run()
#21 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController()
#22 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#23 C:\xampp\htdocs\crediTaxi\index.php(13): CWebApplication->run()
REQUEST_URI=/crediTaxi/site/login
in C:\xampp\htdocs\crediTaxi\protected\views\layouts\main.php (35)
in C:\xampp\htdocs\crediTaxi\protected\views\layouts\column1.php (6)
in C:\xampp\htdocs\crediTaxi\protected\controllers\SiteController.php (67)
2013/12/05 20:38:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "Lavada".' in C:\xampp\htdocs\crediTaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(270): CController->missingAction('Lavada')
#1 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('Lavada')
#2 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('Producto/Lavada')
#3 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/crediTaxi/Producto/Lavada
HTTP_REFERER=http://localhost/crediTaxi/Taxi
---
2013/12/05 20:38:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "Lavada".' in C:\xampp\htdocs\crediTaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(270): CController->missingAction('Lavada')
#1 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('Lavada')
#2 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('Producto/Lavada')
#3 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/crediTaxi/Producto/Lavada
HTTP_REFERER=http://localhost/crediTaxi/Taxi
---
2013/12/06 16:11:24 [error] [system.db.CDbCommand] CDbCommand::execute() falló: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`conductor`, CONSTRAINT `FK_conduce_Taxi` FOREIGN KEY (`placa`) REFERENCES `taxi` (`placa`)). La sentencia SQL ejecutada fue: INSERT INTO `conductor` (`id_conductor`, `placa`) VALUES (:yp0, :yp1).
in C:\xampp\htdocs\creditaxi\protected\controllers\ConductorController.php (73)
in C:\xampp\htdocs\creditaxi\index.php (13)
2013/12/06 16:11:24 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falló al ejecutar la sentencia SQL: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`conductor`, CONSTRAINT `FK_conduce_Taxi` FOREIGN KEY (`placa`) REFERENCES `taxi` (`placa`)). The SQL statement executed was: INSERT INTO `conductor` (`id_conductor`, `placa`) VALUES (:yp0, :yp1)' in C:\xampp\htdocs\creditaxi\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\creditaxi\protected\controllers\ConductorController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): ConductorController->actionCreate()
#4 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('conductor/creat...')
#14 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/creditaxi/conductor/create
HTTP_REFERER=http://localhost/creditaxi/conductor/create
---
2013/12/06 17:19:52 [error] [exception.CException] exception 'CException' with message 'Propiedad "Cliente"."2" no se encuentra definida.' in C:\xampp\htdocs\creditaxi\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(143): CComponent->__get('2')
#1 C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('2')
#2 C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Cliente), 2)
#3 C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php(1257): CHtml::activeInputField('text', Object(Cliente), 2, Array)
#4 C:\xampp\htdocs\creditaxi\framework\web\widgets\CActiveForm.php(657): CHtml::activeTextField(Object(Cliente), 2, Array)
#5 C:\xampp\htdocs\creditaxi\protected\views\cliente\_form.php(26): CActiveForm->textField(Object(Cliente), 2)
#6 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\creditaxi\protected\views\cliente\create.php(18): CController->renderPartial('_form', Array)
#10 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\creditaxi\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\creditaxi\protected\controllers\ClienteController.php(79): CController->render('create', Array)
#15 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): ClienteController->actionCreate()
#16 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/create')
#26 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/creditaxi/cliente/create
HTTP_REFERER=http://localhost/creditaxi/Cliente
---
2013/12/06 17:20:11 [error] [exception.CException] exception 'CException' with message 'Propiedad "Cliente"."2" no se encuentra definida.' in C:\xampp\htdocs\creditaxi\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(143): CComponent->__get('2')
#1 C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('2')
#2 C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Cliente), '2')
#3 C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php(1257): CHtml::activeInputField('text', Object(Cliente), '2', Array)
#4 C:\xampp\htdocs\creditaxi\framework\web\widgets\CActiveForm.php(657): CHtml::activeTextField(Object(Cliente), '2', Array)
#5 C:\xampp\htdocs\creditaxi\protected\views\cliente\_form.php(26): CActiveForm->textField(Object(Cliente), '2')
#6 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\creditaxi\protected\views\cliente\create.php(18): CController->renderPartial('_form', Array)
#10 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\creditaxi\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\creditaxi\protected\controllers\ClienteController.php(79): CController->render('create', Array)
#15 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): ClienteController->actionCreate()
#16 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/create')
#26 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/creditaxi/cliente/create
HTTP_REFERER=http://localhost/creditaxi/Cliente
---
2013/12/06 17:21:34 [error] [exception.CException] exception 'CException' with message 'Propiedad "Cliente"."2" no se encuentra definida.' in C:\xampp\htdocs\creditaxi\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(143): CComponent->__get('2')
#1 C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('2')
#2 C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(Cliente), 2)
#3 C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php(1257): CHtml::activeInputField('text', Object(Cliente), 2, Array)
#4 C:\xampp\htdocs\creditaxi\framework\web\widgets\CActiveForm.php(657): CHtml::activeTextField(Object(Cliente), 2, Array)
#5 C:\xampp\htdocs\creditaxi\protected\views\cliente\_form.php(27): CActiveForm->textField(Object(Cliente), 2)
#6 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\creditaxi\protected\views\cliente\create.php(18): CController->renderPartial('_form', Array)
#10 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\creditaxi\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\creditaxi\protected\controllers\ClienteController.php(79): CController->render('create', Array)
#15 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): ClienteController->actionCreate()
#16 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#19 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#23 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('create')
#25 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/create')
#26 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#27 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/creditaxi/cliente/create
HTTP_REFERER=http://localhost/creditaxi/Cliente
---
2013/12/06 17:30:12 [error] [php] Cannot use a scalar value as an array (C:\xampp\htdocs\creditaxi\framework\web\helpers\CHtml.php:2186)
Stack trace:
#0 C:\xampp\htdocs\creditaxi\protected\views\cliente\_form.php(27): CActiveForm->textField()
#1 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require()
#2 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): ClienteController->renderInternal()
#3 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): ClienteController->renderFile()
#4 C:\xampp\htdocs\creditaxi\protected\views\cliente\create.php(18): ClienteController->renderPartial()
#5 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require()
#6 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): ClienteController->renderInternal()
#7 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): ClienteController->renderFile()
#8 C:\xampp\htdocs\creditaxi\framework\web\CController.php(782): ClienteController->renderPartial()
#9 C:\xampp\htdocs\creditaxi\protected\controllers\ClienteController.php(79): ClienteController->render()
#10 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): ClienteController->actionCreate()
#11 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams()
#12 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): ClienteController->runAction()
#13 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CAccessControlFilter->filter()
#15 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): ClienteController->filterAccessControl()
#16 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#17 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): ClienteController->runActionWithFilters()
#19 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): ClienteController->run()
#20 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController()
#21 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\creditaxi\index.php(13): CWebApplication->run()
REQUEST_URI=/creditaxi/cliente/create
in C:\xampp\htdocs\creditaxi\protected\views\cliente\_form.php (27)
in C:\xampp\htdocs\creditaxi\protected\views\cliente\create.php (18)
in C:\xampp\htdocs\creditaxi\protected\controllers\ClienteController.php (79)
2013/12/06 18:14:47 [error] [system.db.CDbCommand] CDbCommand::execute() falló: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`cliente`, CONSTRAINT `cliente_ibfk_1` FOREIGN KEY (`id_cliente`) REFERENCES `usuario` (`idUsuario`) ON DELETE CASCADE ON UPDATE CASCADE). La sentencia SQL ejecutada fue: INSERT INTO `cliente` (`id_cliente`, `id_rol`, `nombre`, `telefono`, `correo`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\creditaxi\protected\controllers\ClienteController.php (73)
in C:\xampp\htdocs\creditaxi\index.php (13)
2013/12/06 18:14:47 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falló al ejecutar la sentencia SQL: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`cliente`, CONSTRAINT `cliente_ibfk_1` FOREIGN KEY (`id_cliente`) REFERENCES `usuario` (`idUsuario`) ON DELETE CASCADE ON UPDATE CASCADE). The SQL statement executed was: INSERT INTO `cliente` (`id_cliente`, `id_rol`, `nombre`, `telefono`, `correo`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\creditaxi\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\creditaxi\protected\controllers\ClienteController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): ClienteController->actionCreate()
#4 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/create')
#14 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/creditaxi/cliente/create
HTTP_REFERER=http://localhost/creditaxi/cliente/create
---
2013/12/06 18:22:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "sobrecupo" solicitada.' in C:\xampp\htdocs\creditaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CController.php(270): CController->missingAction('sobrecupo')
#1 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('sobrecupo')
#2 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('Taxi/sobrecupo')
#3 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/creditaxi/Taxi/sobrecupo
HTTP_REFERER=http://localhost/creditaxi/Cliente
---
2013/12/06 18:23:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "sobrecupo" solicitada.' in C:\xampp\htdocs\creditaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CController.php(270): CController->missingAction('sobrecupo')
#1 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('sobrecupo')
#2 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('Taxi/sobrecupo')
#3 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/creditaxi/Taxi/sobrecupo
HTTP_REFERER=http://localhost/creditaxi/Cliente
---
2013/12/06 18:26:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "sobrecupo" solicitada.' in C:\xampp\htdocs\creditaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CController.php(270): CController->missingAction('sobrecupo')
#1 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('sobrecupo')
#2 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('Taxi/sobrecupo')
#3 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/creditaxi/Taxi/sobrecupo
HTTP_REFERER=http://localhost/creditaxi/Cliente
---
2013/12/06 18:31:43 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/delete')
#9 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/creditaxi/cliente/delete/12345
HTTP_REFERER=http://localhost/creditaxi/cliente/12345
---
2013/12/06 18:32:12 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/delete')
#9 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/creditaxi/cliente/delete/12345
HTTP_REFERER=http://localhost/creditaxi/cliente/12345
---
2013/12/06 18:33:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\creditaxi\protected\controllers\ClienteController.php:157
Stack trace:
#0 C:\xampp\htdocs\creditaxi\protected\controllers\ClienteController.php(55): ClienteController->loadModel('12345')
#1 [internal function]: ClienteController->actionView('12345')
#2 C:\xampp\htdocs\creditaxi\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(ClienteController), Array)
#3 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ClienteController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/view')
#14 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/creditaxi/cliente/12345
HTTP_REFERER=http://localhost/creditaxi/Cliente
---
2013/12/06 19:04:29 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('Taxi/admin')
#9 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/creditaxi/Taxi/admin
HTTP_REFERER=http://localhost/creditaxi/Cliente
---
2013/12/06 19:04:32 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('Taxi/admin')
#9 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/creditaxi/Taxi/admin
HTTP_REFERER=http://localhost/creditaxi/Taxi/admin
---
2013/12/06 19:07:13 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('Taxi/admin')
#9 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/creditaxi/Taxi/admin
HTTP_REFERER=http://localhost/creditaxi/cliente/12345
---
2013/12/06 19:09:01 [error] [system.db.CDbCommand] CDbCommand::execute() falló: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`conductor`, CONSTRAINT `FK_conduce_Taxi` FOREIGN KEY (`placa`) REFERENCES `taxi` (`placa`)). La sentencia SQL ejecutada fue: INSERT INTO `conductor` (`id_conductor`, `placa`) VALUES (:yp0, :yp1).
in C:\xampp\htdocs\creditaxi\protected\controllers\ConductorController.php (73)
in C:\xampp\htdocs\creditaxi\index.php (13)
2013/12/06 19:09:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falló al ejecutar la sentencia SQL: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`conductor`, CONSTRAINT `FK_conduce_Taxi` FOREIGN KEY (`placa`) REFERENCES `taxi` (`placa`)). The SQL statement executed was: INSERT INTO `conductor` (`id_conductor`, `placa`) VALUES (:yp0, :yp1)' in C:\xampp\htdocs\creditaxi\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\creditaxi\protected\controllers\ConductorController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): ConductorController->actionCreate()
#4 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('conductor/creat...')
#14 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/creditaxi/conductor/create
HTTP_REFERER=http://localhost/creditaxi/conductor/create
---
2013/12/06 19:28:01 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('flota/admin')
#9 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/creditaxi/flota/admin
HTTP_REFERER=http://localhost/creditaxi/taxi/create
---
2013/12/06 19:37:51 [error] [system.db.CDbCommand] CDbCommand::execute() falló: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '12345' for key 'UQ_Flota_id_cliente'. La sentencia SQL ejecutada fue: INSERT INTO `flota` (`id_cliente`, `sobrecupoAgotado`) VALUES (:yp0, :yp1).
in C:\xampp\htdocs\creditaxi\protected\controllers\FlotaController.php (72)
in C:\xampp\htdocs\creditaxi\index.php (13)
2013/12/06 19:37:51 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falló al ejecutar la sentencia SQL: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '12345' for key 'UQ_Flota_id_cliente'. The SQL statement executed was: INSERT INTO `flota` (`id_cliente`, `sobrecupoAgotado`) VALUES (:yp0, :yp1)' in C:\xampp\htdocs\creditaxi\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\creditaxi\protected\controllers\FlotaController.php(72): CActiveRecord->save()
#3 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): FlotaController->actionCreate()
#4 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('flota/create')
#14 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/creditaxi/flota/create
HTTP_REFERER=http://localhost/creditaxi/flota/create
---
2013/12/06 19:47:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "CAU123" solicitada.' in C:\xampp\htdocs\creditaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CController.php(270): CController->missingAction('CAU123')
#1 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('CAU123')
#2 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('taxi/CAU123')
#3 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/creditaxi/taxi/CAU123
HTTP_REFERER=http://localhost/creditaxi/taxi/create
---
2013/12/06 19:48:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "CAU123" solicitada.' in C:\xampp\htdocs\creditaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CController.php(270): CController->missingAction('CAU123')
#1 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('CAU123')
#2 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('taxi/CAU123')
#3 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/creditaxi/taxi/CAU123
HTTP_REFERER=http://localhost/creditaxi/taxi/index
---
2013/12/06 19:58:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "Combustible" solicitada.' in C:\xampp\htdocs\creditaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CController.php(270): CController->missingAction('Combustible')
#1 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('Combustible')
#2 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('Producto/Combus...')
#3 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/creditaxi/Producto/Combustible
HTTP_REFERER=http://localhost/creditaxi/taxi/index
---
2013/12/06 20:11:38 [error] [exception.CException] exception 'CException' with message 'Propiedad "Conductor"."id_cliente" no se encuentra definida.' in C:\xampp\htdocs\creditaxi\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id_cliente')
#1 C:\xampp\htdocs\creditaxi\protected\views\conductor\_form.php(27): CActiveRecord->__get('id_cliente')
#2 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\creditaxi\protected\views\conductor\create.php(18): CController->renderPartial('_form', Array)
#6 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\creditaxi\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 C:\xampp\htdocs\creditaxi\protected\controllers\ConductorController.php(79): CController->render('create', Array)
#11 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): ConductorController->actionCreate()
#12 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#19 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('create')
#21 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('conductor/creat...')
#22 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#23 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/creditaxi/conductor/create
HTTP_REFERER=http://localhost/creditaxi/Conductor
---
2013/12/06 20:12:07 [error] [exception.CException] exception 'CException' with message 'Propiedad "Conductor"."id_cliente" no se encuentra definida.' in C:\xampp\htdocs\creditaxi\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id_cliente')
#1 C:\xampp\htdocs\creditaxi\protected\views\conductor\_form.php(27): CActiveRecord->__get('id_cliente')
#2 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\creditaxi\protected\views\conductor\create.php(18): CController->renderPartial('_form', Array)
#6 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\creditaxi\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\creditaxi\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\creditaxi\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 C:\xampp\htdocs\creditaxi\protected\controllers\ConductorController.php(79): CController->render('create', Array)
#11 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): ConductorController->actionCreate()
#12 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 C:\xampp\htdocs\creditaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\creditaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 C:\xampp\htdocs\creditaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\creditaxi\framework\web\CController.php(291): CFilterChain->run()
#19 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('create')
#21 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('conductor/creat...')
#22 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#23 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/creditaxi/conductor/create
HTTP_REFERER=http://localhost/creditaxi/Conductor
---
